### PR TITLE
fix account creation

### PR DESCRIPTION
**Pull request by jamhed**

### CODE REVIEW

**Comment by lazedo** [comment]
@jamhed hey, is there a jira ticket for this ? if not, what is it fixing ?

**Comment by jamhed** [comment]
@lazedo, it is impossible to create accounts without this fix. also see https://github.com/2600hz/kazoo/pull/2643/files

**Comment by icehess** [comment]
@lazedo @jamhed sorry this was my mistake in previous pr, please review and merge #2643 and #2644 to fix this

**Comment by lazedo** [comment]
it seems to me that we are trying to fix the same thing., fyi i just created one account.

**Comment by jamhed** [comment]
@lazedo, you can't create even the first account on clean install.

**Comment by lazedo** [comment]
sorry if i didn't explained well, i just created an account in a running environment

File: core/kazoo_modb/src/kazoo_modb.erl [hunk 1]
```diff
@@ -272,11 +272,12 @@ maybe_create_current_modb(<<"account%2F", AccountId/binary>>) ->
     maybe_create_current_modb(binary:replace(AccountId, <<"%2F">>, <<>>, ['global'])).
 
 -spec create(ne_binary()) -> 'ok'.
-create(?MATCH_MODB_SUFFIX_RAW(AccountId, _, _) = AccountMODb) ->
```
Comment (lazedo) [inline core/kazoo_modb/src/kazoo_modb.erl]: i don't like this, as in #2643 and #2644 , if the problem is encoded modb name, add a function clause.

**Comment by lazedo** [comment]
@jamhed  the builds do create the first account
```
14:29:52.593 [warning] |webhooks_init|webhooks_util:551 (<0.4831.0>) master account not available
14:29:52.596 [warning] |webhooks_init|webhooks_util:551 (<0.4831.0>) master account not available
14:29:52.599 [warning] |webhooks_init|webhooks_util:551 (<0.4831.0>) master account not available
created new account '6ba7d69d2e6786ffe0c5b7a490182d24' in db 'account%2F6b%2Fa7%2Fd69d2e6786ffe0c5b7a490182d24'
created new account admin user '26c8f3668cd003c9b4c21328b669e424'
promoting account 6ba7d69d2e6786ffe0c5b7a490182d24 to reseller status, updating sub accounts
updating master account id in system_config.accounts
```

**Comment by jamhed** [comment]
@lazedo, well, i can't create account using crossbar_maintenance:create. please tell me how to create master account.

**Comment by lazedo** [comment]
check the builds

**Comment by lazedo** [comment]
i meant the circleci build logs when it starts the application. i believe there's a script for this.

**Comment by jamhed** [comment]
@lazedo, I've checked, script in circle-ci does exactly the same call to crossbar_maintenance:create/4.

**Comment by lazedo** [comment]
@jamhed this is related to #2635

**Comment by jamhed** [comment]
@lazedo, thanks, i know. and @icehess did the fix.

**Pull request closed without merging**